code/chacha20poly1305/Hacl.Impl.Chacha20Poly1305.fst

```diff
@@ -201,12 +201,12 @@ val aead_decrypt: #w:field_spec -> aead_decrypt_st w
 [@ Meta.Attribute.specialize ]
 let aead_decrypt #w k n aadlen aad mlen m cipher mac =
   push_frame();
-  let h0 = get() in
+  let h0 = ST.get() in
```

Comment:
I would much prefer `HS` than `ST` here. But I agree that having the prefix is better.

Reply:
I can do that but I used `ST` because many people write `ST.get` (some files had a mix of `ST.get` and `get` actually), and  the `module ST = FStar.HyperStack.ST` abbreviation is often present in the files.

Reply:
Also, the reason I added this prefix is that I introduced a `Lib.Buffer.get` definition, which shadows those occurrences of `get`

Reply:
Actually, ideally we want to use `Lib.Buffer.get`